content/en/community/code-of-conduct.md

```diff
@@ -6,7 +6,7 @@ description: |
   to establish a safe, respectful and inclusive environment.
 ---
 
-# Kubernetes Community Code of Conduct
+# CNCF Community Code of Conduct v1.3
```

Comment:
I would leave this header intact. It _is_ the code of conduct for Kubernetes, even if we `#include <cncf/coc.h>` to make our lives easier.

Reply:
Done, thanks for your suggestion